src/patternfly/layouts/Grid/grid-order.scss

```diff
@@ -0,0 +1,25 @@
+$pf-l-grid--order--limit: $pf-global--order--limit;
```

Comment:
It seems like the grid could be built from like `$pf-l-grid--columns/tracks: 12;` and both the grid and ordering could reference that, so that the order limit always matches the number of tracks - versus a var for one and a hard-coded value for the other?